utils/python/CIME/SystemTests/eri.py

```diff
@@ -90,6 +90,9 @@ def run_phase(self):
         clone1.set_value("REST_N", rest_n1)
         clone1.set_value("HIST_OPTION", "never")
         clone1.flush()
+        # if the initial case is hybrid this will put the reference data in the correct location
+        if orig_runtype == "hybrid":
```

Comment:
Why do you only do this for hybrid? Wouldn't it also be needed for branch?

But more generally: If I'm understanding the check_all_input_data call correctly, then it looks like it would work to just call that all the time - i.e., removing the check of this orig_runtype. Then it already does its own check for whether any work is needed:

```python
    if get_refcase and run_type != "startup" and not continue_run:
```